icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -86,7 +87,9 @@ void UnitsTest::testConversionCapability() {
         ConversionRates conversionRates(status);
         auto convertibility = checkConvertibility(source, target, conversionRates, status);
 
-        assertEquals("Conversion Capability", testCase.expectedState, convertibility);
+        assertEquals(UnicodeString("Conversion Capability: ") + testCase.source + " to " +
```

Comment:
Very Optional: use `US_INV` constructor for UnicodeString.  For example, `UnicodeString(char*, -1, US_INV)`

Reply:
Leaving it, for the sake of readability: I'm also only using UnicodeString because it has the nice operator+, generally a little bit wasteful...